crates/web/frontend/next.config.mjs

```diff
@@ -1,4 +1,14 @@
+const apiEndpoint = process.env.API_ENDOINT || "http://localhost:3000";
```

Comment:
In the database url of the backend I named the variable `DATABASE_URL`

should we change it to `DATABASE_ENDPOINT`? or this one to `API_URL` to be consistent?